src/main/scala/PatternMatching.scala

```diff
@@ -38,7 +38,7 @@ object PatternMatching {
       def unapplySeq(name: String): Option[Seq[String]] = {
         val names = name.trim.split(" ")
         if (names.size < 2) None
-        else Some(names.last :: names.head :: names.drop(1).dropRight(1).toList)
+        else                Some(names.last :: names.head :: names.drop(1).dropRight(1).toList)
```

Comment:
is this intentional?

Reply:
Yes. I try to align the results of each branch, here they are `None` and `Some`, to improve readability.

Reply:
That does not follow our coding style. We should keep it as it was.

Reply:
> That does not follow our coding style. We should keep it as it was.
OK. I can revert this change.

Reply:
> That does not follow our coding style. We should keep it as it was.

OK. I can revert this change.